src/Widgets/NavigationWidget/SubComponents/CurrentUserDropdown.tsx

```diff
@@ -77,6 +80,32 @@ const LogOutButton = connect(function LogOutButton({
 }: {
   logOutLabel: string
 }) {
+  // TODO: Remove workaround, once #10276 is available
+  if (isEditorLoggedIn()) {
+    return (
+      <OverlayTrigger
+        placement="left"
+        overlay={
+          <Tooltip>
+            Logging out from an app inside the Scrivito UI is currently not
+            possible.
+          </Tooltip>
+        }
+      >
+        <div>
+          <NavDropdown.Item
+            eventKey="MetaNavigation-LogOut"
+            key="MetaNavigation-LogOut"
+            disabled
+          >
+            <i className={`bi bi-box-arrow-right`}></i>
+            {logOutLabel}
```

Comment:
FYI IMHO the button label is barely readable. Please increase the contrast.

<img width="390" alt="image" src="https://github.com/Scrivito/scrivito-portal-app/assets/115093421/62822d9c-36ec-4f15-b8c7-3b8e19926f90">